app/src/providers/stable-diffusion/client.ts

```diff
@@ -0,0 +1,49 @@
+import axios, { Axios } from "axios";
+import { PromptCreationRequestBody, StableDiffusionConfigOptions } from "./client.types";
+import { Prediction } from "replicate";
+
+export class StableDiffusionClient {
+  private readonly httpClient: Axios | undefined = undefined;
+  private readonly clientConfig: StableDiffusionConfigOptions;
+
+  constructor(clientConfig?: StableDiffusionConfigOptions) {
+    this.clientConfig = clientConfig ?? {
+      baseURL: "https://api.replicate.com/v1",
+      auth: process.env.REPLICATE_API_TOKEN ?? "",
+      modelVersion: process.env.REPLICATE_MODEL ?? "",
+    };
```

Comment:
If we are to create and use this client, let's implement it in this part: https://github.com/aufacicenta/pulsemarkets/blob/prompt-wars/app/src/pages/api/prompt-wars/reveal.ts#L59-L64 — preferably the `model` string should be set in this `StableDiffusionClient` or in an env var.